alpha/template/composite/composite_test.go

```diff
@@ -589,10 +591,21 @@ func TestFetchCatalogConfig(t *testing.T) {
 				require.Equal(t, "opening catalog config file \"file/test.yaml\": open file/test.yaml: no such file or directory", err.Error())
 			},
 		},
+		{
+			name: "Successful absolute reference file fetch",
+			path: testDir + "/file/test.yaml",
+			fakeGetter: &fakeGetter{
+				catalog:     validCatalog,
+				shouldError: true,
+			},
+			createFile: true,
+			assertions: func(t *testing.T, rc io.ReadCloser, err error) {
+				require.NoError(t, err)
+				require.NotNil(t, rc)
+			},
+		},
```

Comment:
I'm actually not sure this whole test case is necessary. Looking at lines 612 - 621 it looks like if we end up creating the file we actually set the file path provided to the `FetchCatalogConfig` function to be the absolute path. This makes me think that we should add `shouldError: true` under https://github.com/operator-framework/operator-registry/pull/1128/files#diff-0299f965ef1d5dc96c54539eccbbcbf7a35e5f38255886bcbb6f8c39edf3364fR574 and that would meet this criteria.

I think the "Failed file fetch" actually tests the relative path case by ensuring it attempts to get a relative path that doesn't exist and errors out.